index.html

```diff
@@ -0,0 +1,17 @@
+---
+---
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="chrome=1">
+  <title>API documentation - CFPB Open Tech</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta
+    http-equiv="refresh"
+    content="0;URL='/api/ccdb'"
+  />
+  <meta name="robots" content="noindex">
+</head>
+<body></body>
```

Comment:
Should we include the fallback message here, just in case the meta refresh redirect fails for some reason?